src/ngraph/axis_vector.hpp

```diff
@@ -51,6 +52,25 @@ namespace ngraph
         NGRAPH_API AxisVector& operator=(AxisVector&& v) noexcept;
     };
 
+    template <>
+    class NGRAPH_API AttributeAdapter<AxisVector> : public ValueAccessor<std::vector<int64_t>>
```

Comment:
Why not using `IndirectVectorValueAccessor` ?

Reply:
Good idea.

Reply:
Done.